CHANGELOG.md

```diff
@@ -51,7 +51,7 @@ All notable, unreleased changes to this project will be documented in this file.
   - Implemented new events: `DRAFT_CREATED`, `DRAFT_ADDED_PRODUCTS`, `DRAFT_REMOVED_PRODUCTS`, and `PAYMENT_FAILED`;
   - Implemented new mail events: `TRACKING_UPDATED`, `DIGITAL_LINKS`;
   - Refactored logic of events, it's now easier than ever to add and send events.
-
+- Allow filtering by lists of enums for some fields - #4078 by @korycins
```

Comment:
Align. Can we somehow be more precise what are `some fields` or it's too complicated to put in summarize?